### PR TITLE
Docs: Brave user warning + misc config

### PR DESCRIPTION
**Pull request by msutkowski**
- [x] Adds a notification for brave users to disable shields
- [x] Adds a dev:docs cmd to scripts
- [x] Updates `CONTRIBUTING.md` to reflect current dev process in the monorepo

### Misc notes:
We should reintroduce `lint` and `format` commands to the repo?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,6 @@
   },
   "homepage": "https://redux-toolkit.js.org",
   "directories": {
-    "doc": "docs",
```
Comment (msutkowski) [inline package.json:16]: This was blocking the install of dependencies for docs - so if you cloned this repo, ran `yarn` and then tried to start the docs, it'd error with a missing formik dependency.

**Comment by msutkowski** [comment]
image

image

File: website/src/theme/DocPageWithBraveWarning/index.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+            localStorage.setItem('brave-warning-dismissed', true)
+            setIsBrave(false)
+          }}
+        >
+          Dismiss
+        </button>
+      </div>
+    </div>
+  ) : null
+}
```
Comment (msutkowski) [inline website/src/theme/DocPageWithBraveWarning/index.js:77]: This is the only code that isn't the standard template that was `swizzle`d

**Comment by markerikson** [comment]
Seems reasonable, and neat!

**Pull request merged**